.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,7 @@
+settings:
```

Comment:
Could we perhaps have a top comment with some reference to docs or, failing that, [the GH](https://github.com/canonical/gh-jira-sync-bot)? Just so that future readers know what this is about.

Reply:
That’s a good suggestion, Let me add a top comment with the repo URL.